### ISSUE TITLE
security/openconnect: doesn't exit cleanly

### ISSUE THREAD

**Issue by liweitianux** [DragonFlyBSD/DeltaPorts#928]
OpenConnect starts fine, but doesn't exit cleanly. As shown in the attached image, the default route is messed, showing a strange destination.  OpenConnect itself also gives error message:
```
route: bad address:
ifconfig: ioctl (SIOCAIFADDR): Destination address required
```
In addition, the created `tun0` interface isn't destroyed.

![Screenshot_2020-04-14_21-56-53](https://user-images.githubusercontent.com/686169/79233972-1f19bd80-7e9c-11ea-968d-3dde1a24846b.png)

I'm using OpenConnect 8.05 on DragonFly v5.9.0.155.g05c3f4-DEVELOPMENT as of 2020-Apr-4.

**Comment by liweitianux** [comment]
Well, the issue is caused by [`sysutils/vpnc-scripts`](https://gitlab.com/openconnect/vpnc-scripts/-/blob/master/vpnc-script) that doesn't well support DragonFly BSD...  Need to patch that port to fix this issue.

**Comment by liweitianux** [comment]
Fixed in PR #929 .
